README.md

```diff
@@ -1,38 +1,160 @@
-# Header
+# VTEX Header
 
-Header is a canonical component that any VTEX app can import.
+## Description
+
+The VTEX Header app is a store component used to display logo, minicart, search bar, telemarketing , category menu, menu link and login. This app is used by store theme.
```

Comment:
I think that is better to add a more generic description instead of list all the apps added in this app, because it can be easily deprecated.